src/main/java/org/zalando/pazuzu/feature/FeaturesResource.java

```diff
@@ -24,58 +22,58 @@
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<FeatureDto> listFeatures(
             @RequestParam(required = false, name = "name") String[] featureNames,
-            @RequestParam(required = false, name = "sorted") Integer sorting,
+            @RequestParam(required = false, name = "sorting") Integer sorting,
             @RequestParam(required = false, name = "offset") Integer offset,
             @RequestParam(required = false, name = "limit") Integer limit,
             HttpServletResponse response)
             throws ServiceException {
         if (featureNames == null) {
             if (offset != null && limit != null) {
                 FeaturesWithTotalCount<FeatureDto> featuresTotalCount =
-                        featureService.getFeaturesWithTotalCount(offset, limit, FeatureDto::ofShort);
+                        featureService.getFeaturesWithTotalCount(offset, limit, FeatureDto::of);
                 response.setHeader(X_TOTAL_COUNT, Long.toString(featuresTotalCount.getTotalCount()));
                 response.setHeader("Access-Control-Expose-Headers", X_TOTAL_COUNT);
                 return featuresTotalCount.getFeatures();
             } else {
-                return featureService.listFeatures("", FeatureDto::ofShort);
+                return featureService.listFeatures("", FeatureDto::of);
             }
         }
         Set<Feature> featureSet = featureService.loadFeatures(Arrays.stream(featureNames).collect(Collectors.toList()));
         if (sorting != null && sorting.equals(TOPOLOGICAL_SORT)) {
             List<Feature> features = featureService.getSortedFeatures(featureSet);
-            return features.stream().map(FeatureDto::ofShort).collect(Collectors.toList());
+            return features.stream().map(FeatureDto::of).collect(Collectors.toList());
         }
-        return featureSet.stream().map(FeatureDto::ofShort).collect(Collectors.toList());
+        return featureSet.stream().map(FeatureDto::of).collect(Collectors.toList());
     }
 
     @RolesAllowed({Roles.USER})
     @RequestMapping(method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<FeatureFullDto> createFeature(@RequestBody FeatureToCreateDto value, UriComponentsBuilder uriBuilder) throws ServiceException {
-        FeatureFullDto feature = featureService.createFeature(
-                value.getName(), value.getDockerData(), value.getTestInstruction(), value.getDescription(),
-                value.getDependencies(), value.getTags(), FeatureFullDto::makeFull);
+    public ResponseEntity<FeatureDto> createFeature(@RequestBody FeatureDto value, UriComponentsBuilder uriBuilder) throws ServiceException {
+        FeatureDto feature = featureService.createFeature(
+                value.getMeta().getName(), value.getSnippet(), value.getTestSnippet(), value.getMeta().getDescription(),
```

Comment:
Currently yes because the database layer is not updated yet, there is no fields to store these data right now but I'll create an issue to fix this